models/mart/mrt_order_monthly_report.sql

```diff
@@ -1,15 +1,33 @@
-SELECT 
-	DATE_TRUNC(order_created_at, MONTH) AS report_date,
-    mapping.account_manager,
-    mapping.state,
-    COUNT(DISTINCT order_id) AS total_orders,
-    AVG(total_distinct_items) AS average_total_distinct_items,
-    SUM(sum_feedback_score) / SUM(total_feedbacks) AS average_feedback_score,
-    AVG(total_order_amount) AS average_total_order_amount
-FROM {{ ref('int_sales_database__order') }} AS orders
-LEFT JOIN {{ ref('stg_google_sheets__account_manager_region_mapping') }} as mapping ON orders.user_state = mapping.state
-GROUP BY report_date,
-    account_manager,
-    state
-
-    -- test commit and merge
\ No newline at end of file
+WITH monthly_users_recap AS (
+SELECT DATE_TRUNC(order_date,month) AS order_month,
+COUNT(DISTINCT user_name) AS total_monthly_users
+FROM dataset_fil_rouge.order
+GROUP BY order_month
+ORDER BY total_monthly_users DESC
+), total_monthly_user_from_jawa_timur AS (
+SELECT DATE_TRUNC(order_date,month) AS order_month,
+COUNT(DISTINCT user.user_name) AS total_monthly_users_from_jawa_timur
+FROM dataset_fil_rouge.order AS orders
+LEFT JOIN dataset_fil_rouge.user AS user ON user.user_name =
+orders.user_name
+WHERE user.customer_state LIKE '%JAWA%TIMUR%'
+GROUP BY order_month
+), monthly_orders_recap AS (
+SELECT DATE_TRUNC(order_date,month) AS order_month,
+COUNT(order_id) AS total_monthly_orders
+FROM dataset_fil_rouge.order
+GROUP BY order_month
+), shipping_cost AS (
+SELECT shipping_cost
+FROM dataset_fil_rouge.order_item
+WHERE price > 7000
+)
```

Comment:
```suggestion
)
```
supprimer la requête shipping_cost inutile